src/grid_map_geo.cpp

```diff
@@ -58,20 +58,20 @@
 #include <gdal/ogr_spatialref.h>
 #endif
 
-GridMapGeo::GridMapGeo() {}
+GridMapGeo::GridMapGeo(const std::string frame_id) { frame_id_ = frame_id; }
 
 GridMapGeo::~GridMapGeo() {}
 
-bool GridMapGeo::Load(const std::string &map_path, bool algin_terrain, const std::string color_map_path) {
-  bool loaded = initializeFromGeotiff(map_path, algin_terrain);
+bool GridMapGeo::Load(const std::string &map_path, const std::string color_map_path) {
```

Comment:
```suggestion
bool GridMapGeo::Load(const std::string &map_path, const std::string &color_map_path) {
```

Let's keep them consistent.